frontend/plugins/i18n-head.ts

```diff
@@ -0,0 +1,8 @@
+// Injects the lang attribute in all nuxt <Head></Head> components
```

Comment:
Minor nitpick that we normally do complete sentences for comments that are their own lines (a Python convention I just do these days for everything). No worries whatsoever on it though :) I can get this.